instagram-app/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Comment = props => {
+    return (
+        <div>
+            <p><strong>{props.comment.username}</strong> {props.comment.text}</p>
+        </div>
+    )
+}
+
+Comment.propTypes = {
```

Comment:
Excellent use of PropTypes: ensuring that comments are receiving the required data.